rules-tests/DeadCode/Rector/Assign/RemoveUnusedVariableAssignRector/Fixture/skip_used_args.php.inc

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Rector\Tests\DeadCode\Rector\Assign\RemoveUnusedVariableAssignRector\Fixture;
+
+class SkipUsedArgs
+{
+    public function run()
+    {
+        function (Node $node) use ($expr, &$data) {
```

Comment:
Should $expr be deleted here?

Reply:
This rule removal is remove the `Assign` part, not the variable, so it is kept.

Reply:
Agree. I thought about that the variable is not relevant for the test itself and it might get in the way when the „dead code“ feature learns to drop unnecessary var-captures

Reply:
cleaned up